tests/src/rename.rs

```diff
@@ -59,3 +62,41 @@ fn rename_round_trip() {
         .unwrap();
     assert_eq!(data, inserted);
 }
+
+#[test]
+#[cfg(feature = "mysql")]
+fn rename_round_trip() {
+    use diesel::connection::SimpleConnection;
+    use diesel::insert_into;
+    let data = vec![
+        TestRename {
+            id: 1,
+            renamed: RenameMe::Mod,
+        },
+        TestRename {
+            id: 2,
+            renamed: RenameMe::WithASpace,
+        },
+    ];
+    let connection = get_connection();
+    connection
+        .batch_execute(
+            r#"
+        CREATE TEMPORARY TABLE IF NOT EXISTS test_rename (
+            id SERIAL PRIMARY KEY,
+            renamed enum('mod', 'type', 'with spaces') NOT NULL
+        );
+    "#,
+        )
+        .unwrap();
+    insert_into(test_rename::table)
+        .values(&data)
+        .execute(&connection)
+        .unwrap();
+    let inserted = test_rename::table.load::<TestRename>(&connection).unwrap();
+    assert_eq!(data, inserted);
+}
+
+use diesel;
+
+no_arg_sql_function!(last_insert_id, diesel::types::Bigint);
```

Comment:
Could you add a comment explaining this line?

Reply:
Gone one better and removed it. It gets the ID of the last inserted row for the current thread, which allows getting that object immediately after insert, because Mysql doesn't support `RETURNING` unlike Postgres. In this case we're getting the entire contents of the table, though, so it's not necessary.